content/doc/addons/cellar.md

```diff
@@ -104,8 +104,9 @@ There are several ways to manage your buckets, find in this section a list of op
 
 Some clients allows you to upload files, list them, delete them, etc, like:
 
+- [WinSCP](https://winscp.net)
 - [Cyberduck](https://cyberduck.io)
-- [Filestash](https://filestash.app)
+- [Filestash](https://www.filestash.app/)
```

Comment:
Better prefer alphabetical order for such lists. Maybe we can add some other S3 clients here, such as [MinIO Client (mc)](https://min.io/docs/minio/linux/reference/minio-mc.html), [S3 Browser](https://s3browser.com/) for example

Reply:
Made the change to keep alphabetical order and i added MinIO and S3 browser. Thanks for the suggestion.